content/doc/book/blueocean/activity.adoc

```diff
@@ -19,93 +19,72 @@ endif::[]
-(a square "&#9724;" inside a circle).
-Runs that have been completed can be re-run by clicking the
-"Re-run" symbol (a counter-clockwise arrow "&#8634;").
-The list can be filtered by branch or pull request by clicking on the "branch"
-drop-down in the list header.
-
-This list does not allow Runs to be edited or marked as favorites.
-Those actions can be done from the "<<branches, branches>>" tab.
-
-== Branches
-
-The "Branches" tab shows a list of all branches that have a
-completed or in-progress Run in the current Pipeline.
-Each line in the list corresponds to a branch in source control,
-footnote:scm[https://en.wikipedia.org/wiki/Source_control_management]
-showing
-<<dashboard#pipeline-health, overall health of the branch>> based on recent Runs,
-status of the most recent Run: id number, commit information, duration, and when the Run completed.
-
-image:blueocean/activity/branches.png[Branches Tab of Activity View, role=center]
-
-Clicking on a branch in this list will bring up the
-<<pipeline-run-details#, Pipeline Run Details>>
-for the latest completed or in-progress Run of that branch.
-"In Progress" Runs can be aborted from this list by clicking on the "Stop" symbol
-(a square "&#9724;" inside a circle).
-Pull requests whose latest Run has been completed can be run again by clicking the
-"Play" symbol (a triangle "&#9654;" inside a circle).
-Clicking the "Edit" symbol (similar to a pencil "&#9998;") opens the
-<<pipeline-editor#, pipeline editor>> on the Pipeline for that branch.
-Clicking the "Favorite" symbol (a star outline "&#9734;") adds a branch to the favorites list shown on the
-<<dashboard#favorites, Dashboard's "Favorites" list>> for this user.
-A favorite branch will show a solid star "&#9733;" and clicking it removes
-this branch from the favorites.
-
-== Pull Requests
-
-The "Pull Requests" tab shows a list of all
-Pull Requests for the current Pipeline that have a completed or in-progress Run.
-(Some source control systems call these "Merge Requests", others do not support them at all.)
-Each line in the list corresponds to a pull request in source control, showing
-the status of the most recent Run: id number, commit information, duration, and when the Run completed.
+* *Pipeline Name* - Selecting this displays the <<activity#, default Activity tab>>
+* *Favorites Toggle* - Selecting the favorite icon &#9734; to the right of the Pipeline name, adds a branch to the favorites list shown on the <<dashboard#favorites, dashboard's favorites list>>.
+* *Tabs* (<<activity, Activity>>, <<branches, Branches>> <<pull-requests, Pull Requests>>) - Selecting one of these will navigate to the corresponding information in the Activity view.
+
+=== Activity
+
+The default tab of the Activity view shows a list of the latest completed or in-progress runs.
+Each line represents the <<dashboard#run-status, status>> of a run, run id, commit information, and when the run completed.
```

Comment:
I a not sure if this applies, since it is referring to something in the code, but typically ID is capitalized (according to this dictionary at least), since who knows if they all agree. https://dictionary.cambridge.org/us/dictionary/english/id 

```suggestion
Each line represents the <<dashboard#run-status, status>> of a run, run id, commit information, and when the run completed.
```